File: mon-rdv-backend/src/main/java/sopra/monRdv/rest/dto/PlanningDTO.java
package sopra.monRdv.rest.dto;

import sopra.monRdv.model.Creneau;
import sopra.monRdv.model.Praticien;

import java.util.List;

public class PlanningDTO {
    private Praticien praticien;
    private List<Creneau> creneau;

    public PlanningDTO() {
        super();
    }
}

File: mon-rdv-frontend/src/app/model/planningDTO.ts
import {Creneau} from "./Creneau";
import {Praticien} from "./praticien";

export class PlanningDTO {

  praticien: Praticien;
  creneaux: Array<Creneau>;

  constructor(praticien: Praticien, creneaux: Array<Creneau>) {
    this.praticien = praticien;
    this.creneaux = creneaux;
  }
}
